### PR TITLE
Feature/83 console draw

### PR DESCRIPTION
**Pull request by bartimys1**
#83

### CODE REVIEW

File: HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/ConsoleIO.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            return Console.ReadLine();
+        }
+
+        /// <inheritdoc/>
+        /// set posittion on console
+        public void SetPosition(int x, int y)
+        {
+            Console.SetCursorPosition(x, y);
+        }
```
Comment (INIage) [inline HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/ConsoleIO.cs]: Why this method exist? D It does the same thing that `ConsoleIO.SetCursorPosition`. I think you should delete one of them

File: HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/FigureProvider.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+        /// <inheritdoc/>
+        /// draw verttical line
+        public void VerticalLine(IBoard board)
+        {
+            for (int i = 0; i < board.BoardSizeY - 1; i++)
+            {
+                board.WriteAt('|', board.BoardSizeX / 2, i);
+            }
+        }
```
Comment (INIage) [inline HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/FigureProvider.cs]: Your figure drawed straight on the board, fix it

File: HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/Board.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        /// <inheritdoc/>
+        /// X-axis size
+        public int BoardSizeX { get; set; }
+
+        /// <inheritdoc/>
+        /// Y-axis size
+        public int BoardSizeY { get; set; }
+
+        private readonly IInputOutputDevice console = new ConsoleIO();
```
Comment (INIage) [inline HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/Board.cs]: I think it's nod good thing and you should initiate object in one place than use constructor for make reference

File: HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/ConsoleIO.cs (hunk 1 above)
Comment (bartimys1) [reply HW3/Ivan_Pavlov/ConsoleDraw/ConsoleDraw/Provider/ConsoleIO.cs]: my bad, fix all

**Pull request merged**